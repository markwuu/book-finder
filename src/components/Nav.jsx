import React from "react";
import NavBar from "./UI/NavBar";

const Nav = () => {
  const links = [
    { url: "https://github.com/markwuu/book-finder", name: "Github" },
    { url: "https://reactjs.org/", name: "React" },
    { url: "https://heroku.com/", name: "Heroku" },
    { url: "https://developers.google.com/books/", name: "Google Books API" },
    { url: "https://www.styled-components.com/", name: "styled-components" },
    { url: "https://github.com/axios/axios", name: "axios" }
  ];
  return (
    <React.Fragment>
      <NavBar>
        {links.map((link, i) => {
          return <a href={link.url}>{link.name}</a>;
        })}
      </NavBar>
    </React.Fragment>
  );
};

export default Nav;
