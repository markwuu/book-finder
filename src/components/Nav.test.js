import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";

import Nav from "./Nav";

describe("Nav", () => {
  test("rendering Nav component", () => {
    const wrapper = shallow(<Nav />);

    expect(wrapper).toMatchSnapshot();
  });
});
