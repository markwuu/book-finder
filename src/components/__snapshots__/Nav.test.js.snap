// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav rendering Nav component 1`] = `
<Fragment>
  <styled.div>
    <a
      href="https://github.com/markwuu/book-finder"
      key="0"
    >
      Github
    </a>
    <a
      href="https://reactjs.org/"
      key="1"
    >
      React
    </a>
    <a
      href="https://heroku.com/"
      key="2"
    >
      Heroku
    </a>
    <a
      href="https://developers.google.com/books/"
      key="3"
    >
      Google Books API
    </a>
    <a
      href="https://www.styled-components.com/"
      key="4"
    >
      styled-components
    </a>
    <a
      href="https://github.com/axios/axios"
      key="5"
    >
      axios
    </a>
    <a
      href="https://jestjs.io/"
      key="6"
    >
      Jest
    </a>
    <a
      href="https://github.com/airbnb/enzyme"
      key="7"
    >
      enzyme
    </a>
  </styled.div>
</Fragment>
`;
